cypress/page/payment-step.page.ts

```diff
@@ -5,7 +5,7 @@ class PaymentPage {
 
     constructor() {
       this.payByBankWire = "p.payment_module > a.bankwire";
```

Comment:
this can be simplified, using just the class, since that class is unique and refers to the bank wire payment